assets/js/googlesitekit/datastore/user/dismissed-items.js

```diff
@@ -66,15 +69,17 @@ const baseActions = {
 	 *
 	 * @since 1.37.0
 	 *
-	 * @param {string} slug Item slug to dismiss.
+	 * @param {string} slug             Item slug to dismiss.
+	 * @param {number} expiresInSeconds An integer number of seconds for expiry. 0 denotes permanent dismissal.
 	 * @return {Object} Generator instance.
 	 */
 	dismissItem: createValidatedAction(
-		( slug ) => {
+		( slug, expiresInSeconds = 0 ) => {
```

Comment:
Let's make the second argument an (optional) object of `options`. Simply passing an integer as the second argument won't be obvious what it is when reading this later where it's used. We have many functions that use an object of options/settings so this would be a bit more consistent with our conventions as well. Don't forget that optional parameters should be documented with the respective `@param {type} [name] Optional. Description...` – or in this case `[options.expiresInSeconds]` 🙂

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/2b259f354141d7a53fd9ed478dd75e859c012981